frontend/src/pages/post/Search/SearchPage.jsx

```diff
@@ -0,0 +1,26 @@
+import { useParams } from "react-router-dom"
+import { useState } from 'react';
+import Post from '../../../components/posts/post/PostController'
+import TextField from '@mui/material/InputBase';
+
+const SearchPage = () => {
+    const [searchTerm, setSeachTerm] = useState('')
+
+    return (
+        <div className="Search">
+            <TextField id="standard-basic" label="Standard" variant="standard" size="normal" placeholder="Seach..." onChange={event => {setSeachTerm(event.target.value);}} />
+            <br></br>
+            {Post.filter((val) => {
+            if (searchTerm=="") {
+            return val
+            } else if ((val.content.toLowerCase().includes(searchTerm.toLowerCase())) || (val.author.toLowerCase().includes(searchTerm.toLowerCase()))) {
+            return val
```

Comment:
These returns should be returning true to work with filter

Reply:
was using "val.content.toLowerCase().includes(searchTerm.toLowerCase()" to narrowdown all the posts with containing the keywords. If the searchTerm is nothing it suppose to just return all present posts. Probably this didn't work because the mockdata and back end database was arranged differently I think.